migrations/20180316133403_add_country_code_table.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+use Phinx\Migration\AbstractMigration;
+
+class AddCountryCodeTable extends AbstractMigration
+{
+     /**
+     * Migrate Up.
+     */
+    public function up()
+    {
+        $this->table('country_codes')
+            ->addColumn('country_name', 'string', [
+                'null' => false,
+                'default' => false
+            ])
+            ->addColumn('dial_code', 'string', [
+                'null' => false,
+                'default' => false
+            ])
+            ->addColumn('country_code', 'string', [
+                'null' => false,
+                'default' => false
+            ])
+          ->create();
```

Comment:
country code could be an index, I think. Leaving it up to Will, tho

Reply:
Yeah, we under index now, I'd rather turn off indexes to tune down. It's worth having one here for sure.